lib/toolbox_web/live/package_live.ex

```diff
@@ -72,7 +72,10 @@ defmodule ToolboxWeb.PackageLive do
 
     related_packages_count =
       if package.category do
-        Toolbox.Packages.category_count(package.category)
+        package.category
+        |> Toolbox.Packages.category_count()
+        |> Kernel.-(1)
+        |> max(0)
```

Comment:
```suggestion
  Toolbox.Packages.category_count(package.category) - 1
```

If there is a category, then category count is >= 1  because the current package is there so doing it - 1 would always be >= 0